RouterOS sync plugin (#121)
Ubiquiti-App/UCRM-plugins: plugins/routeros/README.md

# Suspension plugin for Mikrotik RouterOS
This plugin synchronizes list of suspended IP addresses from UCRM to RouterOS based router. 

## Configuration

> To enable redirection to suspension page must be manually set _Filter Rules_ and _NAT_ firewall rules.

Reviewer: ```suggestion
To enable suspension page redirect, you must manually configure _Filter Rules_ and _NAT_ firewall rules.
```

Reviewer: maybe include some manual on how to do this?
